Gems/Atom/Utils/Code/Source/PngFile.cpp

```diff
@@ -68,11 +108,12 @@ namespace AZ
         {
             if (!loadSettings.m_errorHandler)
             {
-                loadSettings.m_errorHandler = [path](const char* message) { DefaultErrorHandler(AZStd::string::format("Could not load file '%s'. %s", path, message).c_str()); };
+                loadSettings.m_errorHandler = [path](const char* message)
+                {
+                    DefaultErrorHandler(AZStd::string::format("Could not load file '%s'. %s", path, message).c_str());
+                };
             }
 
-            // For documentation of this code, see http://www.libpng.org/pub/png/libpng-1.4.0-manual.pdf chapter 3
```

Comment:
Why removing it?  since you didn't provide any other ref I strongly suggest to either keep it or direct to a more concrete reference.